src/WebView.php

```diff
@@ -4,11 +4,12 @@
 
 namespace Yiisoft\View;
 
-use Yiisoft\Html\Html;
 use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Html\Html;
 use Yiisoft\View\Event\BodyBegin;
 use Yiisoft\View\Event\BodyEnd;
 use Yiisoft\View\Event\PageEnd;
+use Yiisoft\Yii\Web\Middleware\Csrf;
```

Comment:
That won't work. There's no yii-web dependency in this package and I'd like not to introduce it: https://github.com/yiisoft/view/blob/master/composer.json#L19